akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sbr/RandomizedBrainResolverIntegrationSpec.scala

```diff
@@ -394,8 +394,11 @@ class RandomizedSplitBrainResolverIntegrationSpec
     def usingLease: Boolean = activeStrategy.contains("lease")
   }
 
+  List(Scenario(leaseMajorityConfig, 3), Scenario(leaseMajorityConfig, 5), Scenario(leaseMajorityConfig, 9))
+  // Scenario(lease-majority, 3) with random seed [1591321960290] in round [1]: cleanSplit [false], healCleanSplit [false] ()flaky [node2 -> (node1, node3); node1 -> (node2); node3 -> (node1, node2)] delays [3, 12, 2, 5, 7, 11, 2, 2, 14, 10]
   val scenarios =
-    List(Scenario(leaseMajorityConfig, 3), Scenario(leaseMajorityConfig, 5), Scenario(leaseMajorityConfig, 9))
+    List(
+      Scenario(ConfigFactory.parseString("""test.random-seed = 1591321960290""").withFallback(leaseMajorityConfig), 3))
```

Comment:
This look really random. How was this generated? 
https://xkcd.com/221/

Reply:
Hehe, up, that was not intended for check in, was exploring another bug fail, good that you caught it